### ISSUE TITLE
DataContext static binding not works

### ISSUE THREAD

**Issue by ydanila** [AvaloniaUI/Avalonia#1007]
I am trying to bind DataContext from markup using this approach.

`<Window xmlns="https://github.com/avaloniaui" MinWidth="500" MinHeight="300"
        xmlns:importer="clr-namespace:Editor.Importer;assembly=Transport.Editor.Importer"
        DataContext="{Binding Source={x:Static importer:ViewModelLocator.Main}}">`

But always on exit from AvaloniaXamlLoader.Load(this) it is empty.

Same time this works like a charm.

`public MainWindow()
 {
   this.InitializeComponent();
   this.AttachDevTools();
   DataContext = ViewModelLocator.Main;
 }`

**Comment by wieslawsoltes** [comment]
Try:
```XAML
DataContext="{Static importer:ViewModelLocator.Main}"
```

**Comment by ydanila** [comment]
Yes. Your approach works. In both cases ViewModelLocator.Main returns correct value, but in my case it doesn't initializes DataContext.

**Comment by jkoritzinsky** [comment]
Can you replicate this on the current nightly version?
